app/utils/forms.tsx

```diff
@@ -191,7 +192,7 @@ export function Button({
 	return (
 		<button
 			{...props}
-			className={clsx(
+			className={twMerge(
 				props.className,
 				getButtonClassName({ size, variant }),
 				'flex justify-center gap-4',
```

Comment:
props.className should be last @kentcdodds

Reply:
You're correct. Mind making a pull request